docs/source/developer_guide/core/internals.rst

```diff
@@ -61,11 +61,11 @@ The :py:class:`~aiida.orm.nodes.Node` can be annotated with labels, description
 
 *Label management:*
 
-- :py:meth:`~aiida.orm.nodes.Node.label` returns the label of the node and can be used as a setter property.
+- :py:attr:`~aiida.orm.nodes.Node.label` returns the label of the node. The setter can be used to *change* the label, e.g. ``mynode.label = "new label"``.
 
 *Description management:*
 
-- :py:meth:`~aiida.orm.nodes.Node.description`: the description of the node (more detailed than the label) and can be used as a setter property.
+- :py:attr:`~aiida.orm.nodes.Node.description`: returns the description of the node (more detailed than the label). The setter can be used to *change* the description.
```

Comment:
Same as above concerning lines per sentence.

Reply:
Again it may be written more "gently"/explicitly:

```suggestion
- :py:attr:`~aiida.orm.nodes.Node.description`: returns the description of the node (more detailed than the label).
  It can also be used to *change* the description, e.g. ``mydnode.description = "new description"``.
```